lib/geordi/gitpt.rb

```diff
@@ -96,4 +119,39 @@ def bold(string)
     HighLine::BOLD + string + HighLine::RESET
   end
 
+  def checkout_branch(name, new_branch: false)
+    if new_branch
+      Geordi::Util.system! 'git', 'checkout', 'master'
+      Geordi::Util.system! 'git', 'checkout', '-b', name
+    else
+      Geordi::Util.system! 'git', 'checkout', name
+    end
+  end
+
+  def normalize_branch_name(name)
```

Comment:
Please call the method `normalize_string` or similar. It is not specific to "branch names" and its only usage is on a _story name_.